tests/src/Interop/Varargs/VarargsNative.cpp

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <xplatform.h>
+#include <stdarg.h>
+
+extern "C" DLL_EXPORT void TestVarArgs(LPWSTR formattedString, SIZE_T bufferSize, LPCWSTR format, ...)
```

Comment:
😄 Thanks @jkoritzinsky I know that. My point is, that isn't clear the calling convention here and since these are tests and we have so many bugs around calling conventions, I am asking why it wasn't marked properly. Perhaps I should have been clearer - Please state the calling convention.

Reply:
:+1: I'll add it!